public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, colorManipulator } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+export default function PubdashFooter() {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.footer}>
+      <span className={styles.logoText}>
+        <a href="https://grafana.com/" target="blank">
+          powered by Grafana <img className={styles.logoImg} src="public/img/grafana_icon.svg"></img>
+        </a>
+      </span>
+    </div>
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  footer: css`
+    position: absolute;
+    height: 30px;
```

Comment:
I Think™️ we're supposed to use spacing units (these may be rem units) or `rem` units instead of `px` 🤔 

I may be wrong tho. Would confirm with frontend peeps.

Reply:
fixed on https://github.com/grafana/grafana/pull/55641/commits/75aefcce079567abadff89e8c0b5553e7f66c123